tasks/bump.js

```diff
@@ -30,6 +33,20 @@ module.exports = function(grunt) {
       versionType: false
     });
 
+    // check if cwd exists
+    if (!fs.existsSync(opts.cwd)) {
+      grunt.fatal('The in "cwd" defined directory does not exist');
+    }
+
+    // set cwd as the working directory for exec
+    var execOpts = {
+      cwd: path.resolve(opts.cwd)
+    };
+
+    // set grunts default working directory
+    grunt.verbose.writeln('Changing working directory: ' + opts.cwd);
+    grunt.file.setBase(opts.cwd);
```

Comment:
I don't know if this sets this just for this task or all tasks so maybe store the current base and then restore it after the tasks have run.